sdk/python/lib/pulumi/resource.py

```diff
@@ -138,7 +138,7 @@ def register_outputs(self, outputs):
             register_resource_outputs(self, outputs)
 
 
-def output(name: str, value: Any):
+def stack_output(name: str, value: Any):
```

Comment:
This rename is because of an unfortunate collision between the `output` module (where the `Output` class is defined). It wasn't possible to export this function from the top-level Pulumi namespace while also having a module named `output`. Since `pulumi.output` means something very different in NodeJS (it's a combinator for turning inputs into outputs), I opted instead to rename the function for exporting stack outputs.

Please help me bikeshed the name if y'all have any better ideas. I also like `stack_export`.

Reply:
Looking at the method, it looks like it actually exports the stack output? If that's the case, do you think `export_stack_output` would be better?

Reply:
I definitely like aligning more closely with our JavaScript SDK, and using the name `export` somehow. Did you consider and reject simply calling it `export`?

Reply:
> Did you consider and reject simply calling it export?

I didn't reject it - I do like `stack_export` as a name since it mirrors the syntax of our JavaScript code. I didn't go with it at first because of the terminology mismatch between the SDK and the CLI: the CLI refers to these as stack outputs, while the JavaScript SDK calls them exports just because of the syntax used to export stuff.

If we want to call them exports, I'm all for that, but we should maybe consider changing our CLI output to call them exports there as well.

Reply:
I'm not a huge fan of `stack_export` because it's not a verb, so feels odd as a function name.

I would be in favor of changing the CLI to match the SDK's terminology (/cc @CyrusNajmabadi). Honestly, I regretted calling this `output` almost immediately after naming it originally.

If we didn't go with just `export` (which still gets my vote -- simpler is better, and it's already qualified by virtue of being part of the Pulumi SDK module), then I like @praneetloke's suggestion, since it is a proper verb and AFAIK Pythonic style often leads to longer names.

Reply:
P. S. If `stack` was a module, and `export` the function name, then I guess it would match the JavaScript SDK almost exactly: `stack.export(..)`.

Reply:
`stack` isn't a top-level module in `pulumi`, unfortunately. I do like just `export`, so I can do that.